projects/go-lib/src/lib/components/go-datepicker/month-view/go-calendar-month-view.component.ts

```diff
@@ -77,13 +77,9 @@ export class GoCalendarMonthViewComponent implements OnChanges, OnInit {
     this.setYear.emit(this.year.value - 1);
   }
 
-  public switchView(): void {
```

Comment:
Was there a reason this was removed?

Reply:
Yes. This was used to switch back from the month view to the day view, but in the DSM there isn't that option, so I removed it